server/handlebars/embed/views/main.hbs

```diff
@@ -8,9 +8,9 @@
     {{/if}}
     {{#unless (dataEmpty testPlanReports)}}
         {{#if (isCandidate phase)}}
-            <details id='embed-report-phase-container'>
-                <summary id='candidate-title'><span>Warning! Unapproved Report</span></summary>
-                <div id='candidate-content-container'>
+            <details class="details">
```

Comment:
While not wrong to have the class name be the same as the element name, and there may be cases why you'd want to do that, I'm not sure why the need here. To me, it seems better to directly target the `details` element rather than define a `.details` class, but this is purely opinion.